src/Components/Dialogs/GetMessage/getMessage.jsx

```diff
@@ -0,0 +1,11 @@
+import getElemen from './GetElement/getElement';
+
+const getItem = (props) => {
+  const { items } = props;
+  const listItems = getElemen(items);
```

Comment:
There was no mentioning of "move X to a separate COMPONENT", there was just "move X to a separate FUNCTION".
Logically, getMessage is a list of elements (a dumb component). So a common solution of creating a dumb list of elements is to create a functional component (as you did with getMessage), then put a mapping result into a separate variable inside or outside of that functional component, and after that, for readability purposes, move map handler into a separate funciton/variable STILL IN THE SAME file as functional component.
All because they are a part of one logical component.

Reply:
I refactored my last code.I understood my mistakes.